spec/index.bs

```diff
@@ -3451,13 +3451,22 @@ A {{GPUProgrammableStageDescriptor}} describes the entry point in the user-provi
             - {{GPUProgrammableStageDescriptor}} |descriptor|
             - {{GPUPipelineLayout}} |layout|
 
-        1. If the |descriptor|.{{GPUProgrammableStageDescriptor/module}} is not
-            a [=valid=] {{GPUShaderModule}} return `false`.
-        1. If the |descriptor|.{{GPUProgrammableStageDescriptor/module}} doesn't contain
-            an entry point at |stage| named |descriptor|.{{GPUProgrammableStageDescriptor/entryPoint}} return `false`.
-        1. For each |binding| that is [=statically used=] by the shader entry point,
-            if the result of [$validating shader binding$](|binding|, |layout|) is `false`, return `false`.
-        1. Return `true`.
+        Return `true` if all of the following conditions are satisfied:
+
+            - The |descriptor|.{{GPUProgrammableStageDescriptor/module}} is [=valid=] {{GPUShaderModule}}.
+            - The |descriptor|.{{GPUProgrammableStageDescriptor/module}} contains
+                an entry point at |stage| named |descriptor|.{{GPUProgrammableStageDescriptor/entryPoint}}.
+            - For each |binding| that is [=statically used=] by the shader entry point,
+                the [$validating shader binding$](|binding|, |layout|) returns `true`.
+            - For any pair of bindings |texture| and |sampler| in |layout|/{{GPUPipelineLayout/[[bindGroupLayouts]]}},
+                one of the following conditions is `false`:
+                - |texture| corresponds to a {{GPUBindGroupLayoutEntry/texture}} binding.
+                - |sampler| corresponds to a {{GPUBindGroupLayoutEntry/sampler}} binding.
+                - |texture| is sampled with |sampler| by the shader.
+                - the sampling call is [=statically used=] by the entry point.
+                - |texture|.{{GPUTextureBindingLayout/sampleType}} is {{GPUTextureSampleType/"unfilterable-float"}}:
```

Comment:
What about int and uint?

Reply:
If the texture is integer/unsigned, then the sampling operation would be considered invalid at `createShaderModule` time, not here. We need that separate section written down, and I'm happy to do that. Just wanted to sync up with @dneto0 first about where this is going to be specified: in the API spec, or WGSL spec.